docker/Dockerfile

```diff
@@ -11,6 +11,7 @@ ARG WEBSERVER_FOLDER=/var/www/html
 COPY --from=front-build-stage /front/dist ${WEBSERVER_FOLDER}
 COPY ./frontend/assets/default_avatar.png ${WEBSERVER_FOLDER}/assets/
 COPY ./frontend/assets/platforms ${WEBSERVER_FOLDER}/assets/platforms
+COPY ./frontend/assets/feed ${WEBSERVER_FOLDER}/assets/feed
```

Comment:
same with the folder structure, changing it to ``frontend/assets/feed/webrcade``?